environments/development/common/rds.tf

```diff
@@ -95,3 +95,38 @@ module "postgres_commodi_tea" {
     customer = "fpo"
   }
 }
+
+module "postgres_admin_aurora" {
+  source = "../../../modules/rds_cluster"
+
+  cluster_name      = "admin-aurora-${var.environment}"
+  engine            = "aurora-postgresql"
+  engine_version    = "13.15"
+  engine_mode       = "provisioned"
+  cluster_instances = 1
+  apply_immediately = true
+
+  instance_class = "db.serverless"
+  database_name  = "PostgresAdmin"
+  username       = "tariff"
+
+  encryption_at_rest = true
+
+  min_capacity = 0.5
+  max_capacity = 2
+
+  security_group_ids = [module.alb-security-group.be_to_rds_security_group_id]
+  private_subnet_ids = data.terraform_remote_state.base.outputs.private_subnet_ids
+
+  tags = {
```

Comment:
I guess it doesn't make any difference if its aurora over standard Postgres :shrug:

Reply:
It'll be cheaper 👀